src/sections/Community/Handbook/repo-data.js

```diff
@@ -41,26 +41,29 @@ export const repo_data = [
         image: meshery,
         site: "https://meshery.io",
         language: "Jekyll",
-        maintainers_name: ["Vacant"],
-        link: ["#"],
+        maintainers_name: ["Nikhil Ladha"],
+        link: ["https://layer5.io/community/members/nikhil-ladha"],
         repository: "https://github.com/meshery/meshery.io",
       },
       {
         project: "Meshery Documentation",
         image: meshery,
         site: "http://docs.meshery.io",
         language: "Jekyll",
-        maintainers_name: ["Vacant"],
-        link: ["#"],
+        maintainers_name: ["Lee Calcote"],
+        link: ["https://layer5.io/community/members/lee-calcote"],
```

Comment:
```suggestion
        maintainers_name: ["Vivek Vishal"],
        link: ["https://layer5.io/community/members/vivek-vishal"],
```

Reply:
@ankushbhardwaj408 will you PR this change in the respective maintainers.md?

Reply:
@leecalcote Should I add maintainers.md to all repos or just to the specific repo you asked me to